src/client.rs

```diff
@@ -2131,6 +2033,18 @@ impl Spotify {
     }
 }
 
+fn join_ids<'a, T: IdType>(ids: impl IntoIterator<Item = Id<'a, T>>) -> String {
```

Comment:
Why not just use `collect` and `join(",")`? We have to allocate anyway with this implementation.